services/workflows-service/src/case-management/types/transition-schema.ts

```diff
@@ -0,0 +1,5 @@
+export interface TransitionSchema {
+  state: string;
+  schema: object;
+  additionalParameters?: object;
```

Comment:
Prefer `Record<PropertyKey, unknown>` over `object`. or `string` instead of `PropertyKey` if its an object received/sent via network.